macos/library/FLEFileChooserPlugin.m

```diff
@@ -30,11 +30,15 @@
 static NSString *const kPlatformMethodNameKey = @"method";
 static NSString *const kPlatformMethodArgsKey = @"args";
 
-// Method argument keys.
+// Method argument keys for NSSavePanel.
+static NSString *const kOKButtonLabelKey = @"prompt";
+static NSString *const kInitialDirectoryKey = @"initialDirectory";
 static NSString *const kAllowedFileTypesKey = @"allowedFileTypes";
+static NSString *const kInitialFileName = @"nameFieldStringValue";
```

Comment:
s/String/Initial/ (or initialFileName for better consistency with initialDirectory). We don't call the other strings "string", and the fact that it's the initial value is important information that the argument name should be capturing.

Reply:
Going with your suggestion of 'initialFileName' for consistency with 'initialDirectory'.